packages/rpc-core/src/rpc-methods/__tests__/get-token-accounts-by-owner-test.ts

```diff
@@ -314,33 +344,40 @@ describe('getTokenAccountsByOwner', () => {
                 )
                 .send();
 
-            await expect(accountInfoPromise).resolves.toMatchObject({
-                value: [
-                    {
-                        account: expect.objectContaining({
-                            data: {
-                                parsed: {
-                                    info: expect.objectContaining({
-                                        isNative: expect.any(Boolean),
-                                        mint: expect.any(String),
-                                        owner,
-                                        state: expect.any(String),
-                                        tokenAmount: {
-                                            amount: expect.any(String),
-                                            decimals: expect.any(Number),
-                                            uiAmount: expect.any(Number),
-                                            uiAmountString: expect.any(String),
-                                        },
-                                    }),
-                                    type: 'account',
+            expect(accountInfos).toMatchObject({
```

Comment:
Oh, why no more `expect(...).resolves.toMatchObject`? When you write it that way, Jest catches and formats exceptions in the case that the promise rejects. When written the way you have it here, the exception just blows up all over the test body itself.